WalletWasabi.Tests/UnitTests/Transactions/TransactionProcessorTests.cs

```diff
@@ -792,7 +792,36 @@ public async Task CorrectSpenderAsync()
 			Assert.Empty(matureTxs);
 
 			Assert.Contains(spentCoin, tx1.WalletInputs);
-			Assert.Contains(spentCoin, tx2.WalletInputs);
```

Comment:
Note TX2 being up to date isn't ensured anymore. TX2 in production is discarded anyway.